src/sass/_markdown.scss

```diff
@@ -131,30 +131,32 @@
     }
   }
 
-  table:not(.lntable) {
-    display: table;
-    border-spacing: 0;
-    border-collapse: collapse;
-    margin-top: $padding-16;
-    margin-bottom: $padding-16;
-    width: 100%;
-    text-align: left;
+  .table-wrap {
+    table:not(.lntable) {
```

Comment:
I would like to do the opposite and use `table:not(.lntable):not(.gist .highlight)`.